testing/python/cpu/test_tilelang_cpu_arch_config.py

```
@@ -0,0 +1,272 @@
+    )
+
+    with pytest.raises(
+        ValueError,
+        match="Conflicting function-level tilelang_pass_configs found in IRModule",
+    ):
+        with tvm.target.Target("c"):
+            tilelang.lower(mod, target="c", target_host="c")
+
+
+def test_lower_rejects_unregistered_pass_config_key():
+    with pytest.raises(Exception):
```

Ruff (0.15.9)

[warning] Do not assert blind exception: `Exception`

(B017)